File: app/src/main/java/com/example/BusService.java
package com.example;

public enum BusService {
    JOTUR(
        "jotur",
        "Jotur",
        "Horários e Mapa Interativo",
        "Linhas metropolitanas integradas de Palhoça, Florianópolis e região. Acesse horários e visualize o mapa interativo de linhas e trajetos.",
        "https://www.jotur.com.br/horarios/",
        true,
        "Com Mapa Interativo",
        0xFF059669
    ),
    CONSORCIO_FENIX(
        "consorcio_fenix",
        "Consórcio Fênix",
        "Horários e Linhas Urbanas",
        "Transporte coletivo de Florianópolis. Consulte quadro de horários completo, partidas em tempo real e itinerários dos ônibus.",
        "https://www.consorciofenix.com.br/horarios",
        false,
        "Florianópolis Urbano",
        0xFFDC2626
    );

    private final String id;
    private final String title;
    private final String subtitle;
    private final String description;
    private final String url;
    private final boolean hasMap;
    private final String tag;
    private final int brandColor;

    BusService(String id, String title, String subtitle, String description, String url, boolean hasMap, String tag, int brandColor) {
        this.id = id;
        this.title = title;
        this.subtitle = subtitle;
        this.description = description;
        this.url = url;
        this.hasMap = hasMap;
        this.tag = tag;
        this.brandColor = brandColor;
    }

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getSubtitle() {
        return subtitle;
    }

    public String getDescription() {
        return description;
    }

    public String getUrl() {
        return url;
    }

    public boolean hasMap() {
        return hasMap;
    }

    public String getTag() {
        return tag;
    }

    public int getBrandColor() {
        return brandColor;
    }

    public static BusService fromId(String id) {
        for (BusService service : values()) {
            if (service.id.equalsIgnoreCase(id)) {
                return service;
            }
        }
        return JOTUR;
    }
}

File: app/src/main/java/com/example/BusService.kt
package com.example

import androidx.compose.ui.graphics.Color
import com.example.ui.theme.FenixRed
import com.example.ui.theme.JoturGreen

enum class BusService(
    val id: String,
    val title: String,
    val subtitle: String,
    val description: String,
    val url: String,
    val hasMap: Boolean,
    val tag: String,
    val brandColor: Color
) {
    JOTUR(
        id = "jotur",
        title = "Jotur",
        subtitle = "Horários e Mapa Interativo",
        description = "Linhas metropolitanas integradas de Palhoça, Florianópolis e região. Acesse horários e visualize o mapa interativo de linhas e trajetos.",
        url = "https://www.jotur.com.br/horarios/",
        hasMap = true,
        tag = "Com Mapa Interativo",
        brandColor = JoturGreen
    ),
    CONSORCIO_FENIX(
        id = "consorcio_fenix",
        title = "Consórcio Fênix",
        subtitle = "Horários e Linhas Urbanas",
        description = "Transporte coletivo de Florianópolis. Consulte quadro de horários completo, partidas em tempo real e itinerários dos ônibus.",
        url = "https://www.consorciofenix.com.br/horarios",
        hasMap = false,
        tag = "Florianópolis Urbano",
        brandColor = FenixRed
    )
}
